### DISCUSSION TITLE
Multiple Constants and Var Declarations

### DISCUSSION THREAD

**Discussion by talley** [dotnet/csharplang#5979] [Language Ideas]
It is possible to allow multiple constants and vars declarations?
---------------------------------------------------------------
Right now you can declare  constant variables like below:
 const int x = 23; 
const int y = 40; 
It will be nice if you can declare them like this below:
const(
     int x=23,
    int y=40
);
---------------------------------------------------------------
Right now you can declare  var variables like below:
var x=23;
var y=40;
It will be nice if you can declare them like this below:
 var(
    x=23,
   y=40
);

Thank You
Talley

**Comment by FaustVX** [comment]
You can use tuple deconstruction to assign to multiple `var`
[`var (x, y, z) = (0, "Hello", new Random())`](https://sharplab.io/#v2:C4LgpgHgDgNghgSwHYBoAmIDUAfAAgJgEYBYAKFwGYACAqgYSoG8yrWbrcAWKgWQAoAlExZtRANzgAnKnwgoqAT3kAvIQF4ZABnkAiABJgYMAPY75SMAHcqAJThI0xgLaCBAbhGsAvmS9A==)

**Comment by koszeggy** [comment]
`const int x = 23, y = 40;` is valid since C# 1.0. Why would you like to add those parentheses?
